tool/gen_mackerel_plugin.pl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+use 5.014;
+use warnings;
+use utf8;
+use autodie;
+use JSON::PP;
+
+my $json = do {
+    local $/;
+    open my $fh, '<', 'packaging/config.json';
+    <$fh>
+};
+
+my @plugins = sort @{ decode_json($json)->{plugins}};
+
+sub resolve_package {
```

Comment:
Currently there's no guarantee that all plugin's package is named in this way.
Should we add some documentations for contributors?